lasagne/utils.py

```diff
@@ -97,6 +97,16 @@ def one_hot(x, m=None):
     Theano tensor variable
         A Theano tensor variable of shape (``n``, `m`), where ``n`` is the
         length of `x`, with the one-hot representation of `x`.
+
+    Notes
+    -----
+
```

Comment:
@mgrachten: I think this empty line is what prevents the notes from showing up in the rendered documentation. You need to stick exactly to [the numpydoc howto](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) to make it work. Do you want to send a PR fixing this?